docs/sphinx/quickstart.rst

```diff
@@ -76,6 +104,19 @@ This indexes a document with the index API:
         },
     )
 
+You can also index multiple documents at once with the bulk helper function:
+
+.. code-block:: python
+
+    from elasticsearch import helpers
+
+    def generate_docs(documents, index_name):
+        for i, document in enumerate(documents):
+            yield dict(_index=index_name, _id=f"{i}", _source=document)
+    
+    helpers.bulk(client, generate_docs(books, index_name))
+
+These helpers are the recommended simple and streamlined way to abstract otherwise complicated and verbose functions such as `client.bulk`.
```

Comment:
Unlike Markdown, you need two backticks in reStructuredText to format as code: single backticks only add emphasis. What do you think of this wording?

```suggestion
These helpers are the recommended way to perform bulk ingestion. While it is also possible to perform bulk ingestion using ``client.bulk`` directly, the helpers handle retries, ingesting chunk by chunk and more. See the :ref:`helpers` page for more details.
```

I've considered linking to the `client.bulk()` docs using `:meth:`~elasticsearch.Elasticsearch.bulk` but I did not find it clearer as it's only rendered as "bulk()" with a link to the docs.